feathr_project/feathr/client.py

```diff
@@ -610,13 +611,50 @@ def monitor_features(self, settings: MonitoringSettings, execution_configuration
         """
         self.materialize_features(settings, execution_configurations, verbose)
 
+    def _get_feature_key(self, feature_name: str):
+        features = []
```

Comment:
Add comments on this method for easier maintenance.

Reply:
nit: better use docstring for comments. givin > given

Reply:
Changed.